lib/slash.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "slash" {
+  function slash(path: string): string;
+  export = slash;
+}
```

Comment:
I believe `slash@3.0` already provides typings. It can be accessed from `./node_modules/slash-BABEL_8_BREAKING-true`.